.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,37 @@
+name: Bug Report
+description: Create a report to help us reproduce and fix the bug
+title: "[Bug]: "
+labels: ["bug"]
+
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Thank you for taking the time to file a bug report.
+        Before creating a new issue, please make sure to take a few minutes to check for already existing issues or information about the bug.
```

Comment:
```suggestion
```

I think only if we get overwhelmed by bug reports should this be done. Right now we are bug report-poor and want to make the barrier for reporting bugs as low as possible.